tests/device_tests/bitcoin/test_signtx_external.py

```
@@ -842,6 +842,49 @@ def test_p2tr_external_unverified(session: Session):
+    out1 = messages.TxOutputType(
+        address="tb1q7r9yvcdgcl6wmtta58yxf29a8kc96jkyxl7y88",
+        amount=15_000,
+        script_type=messages.OutputScriptType.PAYTOADDRESS,
+    )
+    out2 = messages.TxOutputType(
+        address_n=parse_path("m/86h/1h/0h/1/0"),
+        script_type=messages.OutputScriptType.PAYTOTAPROOT,
+        amount=6_800 + 13_000 - 200 - 15_000,
+    )
+
+    with pytest.raises(TrezorFailure, match="[Ee]xternal input"):
```

Ruff (0.15.21)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)